addon/converter/fixture-converter.js

```diff
@@ -135,7 +135,7 @@ export default class {
           attrName = attrOptions;
         }
       }
-      return attrName || keyFn.apply(this, [attribute, method]);
+      return attrName || keyFn.apply(serializer, [attribute, method, 'serialize']);
```

Comment:
This change surprised me the most, as I would have thought that an incorrect `this` within a serializers' `keyForRelationship` / `keyForAttribute` would be something somebody would have stumbled over earlier 🤔 

Passing the 3rd `serialize` argument is needed so we ensure [this path](https://github.com/emberjs/data/blob/v3.0.2/addon/serializers/embedded-records-mixin.js#L135) is taken when the embedded records mixin is used.

Reply:
I recall a year ago I was fumbling with this .. and I just don't remember what happened .. or I how this got to this point ( been too long ) so welcome the fresh eyes on this 👍